pkg/engine/combat/heal.go

```diff
@@ -3,32 +3,66 @@ package combat
 import (
 	"github.com/simimpact/srsim/pkg/engine/event"
 	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/model"
 )
 
 func (mgr *Manager) Heal(heal info.Heal) {
+	source := mgr.attr.Stats(heal.Source)
```

Comment:
`source` creation should be moved inside the for loop. Anything that hooks onto `HealStart` will modify the source stats. If reusing the same stats, an aoe heal will get its stats applied and stored between targets